packages/matchbox/src/components/Expandable/styles.js

```diff
@@ -10,7 +10,7 @@ export const StyledHeader = styled('button')`
   ${() =>
     css({
       px: '450',
-      py: '200',
+      py: '300',
```

Comment:
I think we need to check if `variant === 'borderless'` here, and use `200` for borderless, `300` for regular ones